scripts/smithy.lic

```diff
@@ -0,0 +1,137 @@
+=begin
+
+  Smithy service helper script
+
+  This script will request multiple services from the smithy. It tries to be
+  strict about what it will accept for input and to verify that a blacksmith
+  NPC is present before requesting services.
+
+  Usage:
+
+  ;smithy <count> <crit, damage, or sighting>
+
+  author: Kragdruk
+  version: 1.0
+  tags: util, ccf, duskruin, festival, merchant
+
+=end
+
+module Smithy
+  SERVICE_TYPES = %w[critical damage sighting]
```

Comment:
I haven't ever actually paid for damage weighting/padding or sighting, but I verified that the requesting works the same.